tests/test_sync.py

```diff
@@ -25,7 +29,11 @@ def test_trie_sync(random):
     while len(requests) > 0:
         results = []
         for request in requests:
-            results.append([request.node_key, src_trie.db[request.node_key]])
+            if len(request.node_key) == 32:
+                results.append([request.node_key, src_trie.db[request.node_key]])
+            else:
+                import rlp
+                results.append([request.node_key, rlp.decode(request.node_key)])
```

Comment:
Maybe a comment explaining that when the RLP representation of a node is less than 32 bytes that is stored directly as the node's key and that's why we don't look it up in the trie here?

Reply:
This is accidentally left in.  This code path should no longer happen, removing.